test/tsconfig.json

```diff
@@ -8,20 +8,19 @@
         "outDir": "../build"
     },
     "atom": {
-        "rewriteTsconfig": false
+        "rewriteTsconfig": true
     },
     "filesGlob": [
         "../typings/tsd.d.ts",
-        "./typings/**/*.d.ts",
+        "./typings/tsd.d.ts",
         "../src/**/*.ts",
-        "/*.ts",
+        "./*.ts",
         "./rules/*.ts",
         "./formatters/*.ts"
     ],
     "files": [
         "../typings/tsd.d.ts",
-        "typings/chai/chai.d.ts",
-        "typings/mocha/mocha.d.ts",
+        "./typings/tsd.d.ts",
```

Comment:
yeah, that makes sense actually -- I was trying to standardize on one style of referencing typings. Seems like the other way is better and both src/ and test/ should remove tsd.d.ts